src/core/qgstransaction.h

```diff
@@ -165,6 +171,11 @@ class CORE_EXPORT QgsTransaction : public QObject SIP_ABSTRACT
      */
     void afterRollback();
 
+    /**
+     * Emitted if a sql query is executed and the underlying data is modified
+     */
+    void dirty( const QString &sql );
```

Comment:
To me this sounds more like a getter, what do you think about `dirtied`, `dirtyChanged` or `oldDirtyBastard`?